src/setupTests.ts

```diff
@@ -1,5 +1,6 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
I included here because test dependencies, for me, is always a devDependency.
So need to tell eslint to not consider this file when doing the above check